### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by aboyandhis**
Makes new branch for website

### CODE REVIEW

**Comment by aboyandhis** [comment]
I suppose I will just commit this into master then .gitignore the build folder in all branches but "Documentation".

My reason for this is so that the Website is never part of the Yutes workflow. 

When we are ready to create docs we change branches and work on it in the **Documentation Branch**

My next step is to setup GitHub actions so perform a CI build to GitHub pages anytime we accept a PR in the Documentation branch. 

In summary

- [] New branch for docs website
